src/index.html

```diff
@@ -1,142 +1,146 @@
-
-  <div id="root"></div>
-
-  <noscript>
-    <style type="text/css">
-      body {
-        overflow: hidden;
-      }
-
-      .noscript-overlay {
-        background-color: white;
-        bottom: 0;
-        left: 0;
-        position: absolute;
-        top: 0;
-        right: 0;
-        z-index: 2000;
-      }
-
-      .noscript-contain {
-        height: 200px;
-        left: 50%;
-        margin-left: -300px;
-        margin-top: -100px;
-        position: absolute;
-        top: 50%;
-        width: 600px;
-      }
-
-      .ep-style-wrapper .regular,
-      .ep-style-wrapper p {
-        font-weight: 400;
-      }
-
-      .ep-style-wrapper .font-xl {
-        font-size: 32px;
-        line-height: 36px;
-      }
-    </style>
-    <div class="noscript-overlay">
-      <div class="noscript-contain ep-style-wrapper">
-        <span class="font-xl regular colour-dark-blue margin-s-bottom block" style="display: block;">This website
-          requires
-          JavaScript
-        </span>
-        <p class="font-xs margin-s-bottom">The features of this website require JavasScript to function. Please check
-          your
-          browser settings and turn on JavaScript.</p>
-        <p class="font-xs">If you can't find the setting, try searching the web for your browser name (Chrome, Firefox,
-          etc.)
-          and "turning on JavaScript".
-        </p>
-      </div>
-    </div>
-  </noscript>
-
-  <!-- Modal Window PLaceholder -->
-  <div data-region="modalRegion" class="modal-container"></div>
-
-</body>
-
-</html>
+<!--
+    Copyright © 2018 Elastic Path Software Inc. All rights reserved.
+
+    This is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    This software is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this license. If not, see
+
+        https://www.gnu.org/licenses/
+-->
+
+<!DOCTYPE html>
+<html manifest="manifest.appcache">
+
+<head>
+
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+  <meta name="description" content="Elastic Path&trade; Visual Cortex - App Framework">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <meta name="theme-color" content="#5bc0de" />
+  <meta property="og:image" content="https://s3-us-west-2.amazonaws.com/ep-demo-images/VESTRI_VIRTUAL/VESTRI_MODEL_X_FOR_KIDS.png">
+  <meta property="og:image:type" content="image/png">
+  <meta property="og:image:width" content="1024">
+  <meta property="og:image:height" content="1024">
```

Comment:
As of your latest commit, it looks like these `og:image` properties are no longer necessary

Reply:
we still need it, image option works only for pinterest

Reply:
also for reach sharing in facebook we need manipulate metatags